components/widgets/team_icon/team_icon.scss

```diff
@@ -39,6 +39,7 @@
     text-transform: uppercase;
     transform-origin: center center;
     transition: box-shadow 0.3s, background-color 0.3s, border-color 0.3s;
+    will-change: box-shadow, background-color, border-color;
```

Comment:
Based on https://developer.mozilla.org/en-US/docs/Web/CSS/will-change, this shouldn't be used unless we're certain that this is causing performance issues. Are you sure we want to add this here?

Reply:
Yes, the report for the team icon animation shows that it's causing some delays. Due to our colour and border changes its a bit too expensive for the browser :P

Reply:
Interesting. I guess it is one of the more fancy animations that we have in the UI